src/main/kotlin/de/smartsquare/squit/config/TestIndexer.kt

```diff
@@ -136,15 +136,20 @@ class TestIndexer(private val projectConfig: Config) {
                     acc.withFallback(config).mergeTag(currentPath.fileName.toString())
                 }
             }
-            .let { projectConfig.withFallback(it) }
+            .let {
+                projectConfig
+                    .setTestDir(path)
```

Comment:
Let's call this `withTestDir` since it's a fluent style modification and also to make it similar to the line below.